versions/a-/arrow.json

```
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "8b0c4c21d05a62cc0b626e93149882cd127f83f8",
```

```suggestion
      "git-tree": "cff712c97f67ce2ef9061d823c751f5e209e5838",
```
The x86-windows port build suceeds, but that triplet also checks these entries which are normally managed with `vcpkg x-add-version [--overwrite-version] arrow`.

---

Ah, thanks. I now understand that better (should've been running that as I amended the PR).